lib/compat.php

```diff
@@ -39,3 +39,46 @@ function ppi_gd_compatible() {
 function ppi_json_compatible() {
     return extension_loaded('json');
 }
+
+/**
+ * Check if an individual grant string contains necessary permissions
+ *
+ * @param string $grant
+ * @param string $db
+ * @return boolean
+ */
+function ppi_mysql_grant_compatible($grant, $db) {
+    $escaped = str_replace('_', '\\_', $db);
```

Comment:
probably just over escaping :) - both result in the same string. ill remove the extra slash